File: joseffe-barroso/Java/src/entrada_processamento_saida/Exercise15.java
package entrada_processamento_saida;

import java.util.Scanner;

public class Exercise15 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		double heigth, weight, imc;
		String gender;
		
		System.out.printf("Digite a sua altura em (m): ");
		heigth = scanner.nextDouble();
		
		System.out.printf("Digite o seu peso (em Kg): ");
		weight = scanner.nextDouble();
		
		System.out.printf("Digite \"M\" para Masculino e \"F\" para feminino: ");
		gender = scanner.next();
		
		while (!gender.equals("M") && !gender.equals("F")) {
			System.out.printf("INFORME UM GENÊRO VALIDO. \"M\" para Masculino e \"F\" para feminino: ");
			gender = scanner.next();
		}
		
		imc = weight / (heigth * heigth);
		
		if (gender.equals("M")) {
			if (imc < 20) System.out.printf("Abaixo do peso!");
			if (imc < 25) System.out.printf("Peso ideal!");
			else System.out.printf("Acima do peso!");
		}
		
		if (gender.equals("F")) {
			if (imc < 19) System.out.printf("Abaixo do peso!");
			if (imc < 24) System.out.printf("Peso ideal!");
			else System.out.printf("Acima do peso!");
		}
		
		scanner.close();
	}

}

File: joseffe-barroso/Python/exercise15.py
heigth = float(input('Digite a sua altura em (m): '))
weight = float(input('Digite o seu peso (em Kg): '))
gender = input('Digite "M" para masculino e "F" para feminino: ')

while (gender != 'M' and gender != 'F'):
    gender = input(
        'INFORME UM GENÊRO VALIDO. "M" para Masculino e "F" para feminino: ')

imc = weight / (heigth * heigth)

if (gender == 'M'):
    if (imc < 20):
        print("Abaixo do peso!")
    elif (imc < 25):
        print("Peso ideal!")
    else:
        print("Acima do peso!")

else:
    if (imc < 19):
        print("Abaixo do peso!")
    elif (imc < 24):
        print("Peso ideal!")
    else:
        print("Acima do peso!")
